en/cpp/api_changes.md

```diff
@@ -0,0 +1,216 @@
+# API Changes
+
+This page tracks changes between versions.
+
+It covers both breaking (incompatible) and non-breaking changes.
+
+## Semantic Versioning
+
+MAVSDK follows [semver/Semantic Versioning](https://semver.org/) conventions where as possible.
+
+The version number has the format: **major.minor.patch**.
+Individual parts of the number are iterated when the:
+
+- **major**: API is changed, or functionality is removed.
+- **minor**: API is extended, functionality is added.
+- **patch**: API is not changed, functionality is not changed, but a bug is fixed.
+
+This means that breaking changes to the API result in a bump of the major version number (e.g. 1.4.3 to 2.0.0).
+
+> **Note** At time of writing, breaking/incompatible changes *are not* resulting in the major version number being increased!
+  We plan to release version 1.0.0 in the near future, after which the above strategy will be adopted (i.e. currently breaking changes can occur in both minor and patch releases).
+
+
+## v0.25.0
+
+There are several changes in this release (from v0.24.0) because the C++ plugins are now partially auto-generated from the proto files. While this causes some painful changes, it has several advantages:
+- Better consistency between C++ and other languages for naming/docs/functionality.
+- Missing features for language wrappers such as Python.
```

Comment:
"Missing features" does not sound like an advantage to me. What about something like:

```suggestion
- Automatically propagate features to language wrappers such as Python.
```